Add blog post entry for kotlin multiplatform

fractalwrench opened the pull request:
## Goal

Adds a blog post introducing a Kotlin Multiplatform implementation of OpenTelemetry as suggested in #7809.

fractalwrench commented:
Thanks for the feedback @svrnm - I've updated the PR with the suggestions and added a couple of sentences about how opentelemetry-java can support Kotlin in certain contexts.

mx-psi commented on content/en/blog/2025/kotlin-multiplatform-opentelemetry.md:
```diff
@@ -0,0 +1,47 @@
+[donation proposal](https://github.com/open-telemetry/community/issues/2975).
+
+If you don't want to become a contributor but _do_ want to give feedback on the
+project so far or try it out, please checkout the
+[repository here](https://github.com/embrace-io/opentelemetry-kotlin).
```
Could we make it more explicit how to give feedback about the repository? (e.g. by encouraging people to file an issue?)

svrnm commented on content/en/blog/2025/kotlin-multiplatform-opentelemetry.md:
```diff
@@ -0,0 +1,48 @@
+---
+title: 'Call for Contributors: OpenTelemetry for Kotlin'
+linkTitle: 'CfC: OTel for Kotlin'
+date: 2025-09-19
```
```suggestion
date: 2025-09-25
```

let's make sure we update the date on publication

svrnm commented:
@fractalwrench you need to update the date and ensure that the PR merges, as you have raised it from the embrace org and not your personal repo, that's why we normally recommend to not do that.

fractalwrench commented:
Thanks @svrnm - I've updated the date to today and rebased the PR. I don't have permissions to merge so assume that'd be yourself or one of the other maintainers?

fractalwrench commented:
@svrnm rebased
